.packer/win/scripts/vcpkg.ps1

```diff
@@ -0,0 +1,19 @@
+$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';
+
+echo "Cloning and setting up vcpkg"
+git clone https://github.com/Microsoft/vcpkg.git C:\vcpkg
+
+echo "Run bootstrap-vcpkg.bat"
+C:\vcpkg\bootstrap-vcpkg.bat
+
+echo "Run vcpkg.exe integrate install"
+C:\vcpkg\vcpkg.exe integrate install
+
+echo "Checkout to commit"
+git -C C:\vcpkg checkout  (Get-Content -Path c:\Windows\Temp\vcpkg\VCPKG_COMMIT_SHA)
```

Comment:
I would move "Checkout to commit" and "Apply patches to vcpkg" before "Run bootstrap-vcpkg.bat", so that commit and patches are also used for vcpkg executable.

Also
```suggestion
git -C C:\vcpkg checkout (Get-Content -Path c:\Windows\Temp\vcpkg\VCPKG_COMMIT_SHA)
```